hack/make/binary-daemon

```diff
@@ -22,6 +22,12 @@ copy_binaries() {
 	done
 }
 
+
+# libdm_dlsym_deferred_remove doesn't make sense for static binaries. If we are
+# using libdm_dlsym_deferred_remove then the current libdevmapper is too old to
+# have deferred removal, so just disable it.
+export BUILDFLAGS=( "${BUILDFLAGS[@]/libdm_dlsym_deferred_remove/libdm_no_deferred_remove}" )
```

Comment:
This no longer make sense as we do not build libdevmapper for `static_build` case.